queries.md

```diff
@@ -5,28 +5,59 @@
 ## 1. Find all the companies that include 'Facebook' on the **name** field.
 
  - **`query`**: {name: 'Facebook'}
- 
+ { name: "Facebook" }
+
  ## 2. Find all the companies which **category_code** is 'web'. Retrive only their `name` field:
 
  - **`query`**: {category_code: 'web'}
+
  - **`projection`**: {name: 1, _id: 0}
 
 ## 3. Find all the companies named "Twitter", and retrieve only their `name`, `category_code` and `founded_year` fields.
+{name: 'Twitter'}
+{name: 1, category_code: 1, founded_year: 1, _id: 0}
 
 ## 4. Find all the companies who have `web` as their **category_code**, but limit the search to 50 companies.
+{category_code: "web"}
+limit: 50
 
 ## 5. Find all the companies which **category_code** is 'enterprise' and have been founded in 2005. Retrieve only the `name`, `category_code` and `founded_year` fields.
+{$and: [ { category_code: "enterprise" }, {founded_year: 2005} ] }
+{name: 1, category_code: 1, founded_year: 1, _id: 0}
 
 ## 6. Find all the companies that have been **founded** on the 2000 or have 20 **employees**. Sort them descendingly by their `number_of_employees`.
 
+{$or: [ { founded_year: 2000 }, {number_of_employees: 20 }]}
+{number_of_employees: -1}
+
 ## 7. Find all the companies that do not include `web` nor `social` on their **category_code**. Limit the search to 20 documents and retrieve only their `name` and `category_code`.
 
+{category_code: { $nin: [ "web", "social"] }}
+{name: 1, category_code: 1}
+limit: 20
+
 ## 8. Find all the companies that were not **founded** on 'June'. Skip the first 50 results and retrieve only the `founded_month` and `name` fields.
 
-## 9. Find all the companies that have 50 employees, but do not correspond to the 'web' **category_code**. 
+{founded_month: {$ne: 6 }}
+{founded_month: 1, name: 1}
```

Comment:
Here you need to specify "_id: 0", otherwise it's shown by default. You did it before so I believe it's just an oversight :)